pkg/operator/ceph/csi/spec.go

```diff
@@ -336,6 +337,10 @@ func (r *ReconcileCSI) startDrivers(ver *version.Info, ownerInfo *k8sutil.OwnerI
 		}
 	}
 
+	if AllowUnsupported || v.SupportsCustomCephConf() {
+		tp.MountCustomCephConf = true
```

Comment:
```suggestion
		tp.MountCustomCephConf = v.SupportsCustomCephConf()
```

Looks like `AllowUnsupported` is tested in `SupportsCustomCephConf` already.